robotd/devices.py

```diff
@@ -128,28 +130,43 @@ def status(self):
         return {'temperature': temp_milli_degrees / 1000}
 
 
+class NoZoneFound(ValueError):
+    """Indicates that a search for a zone file failed."""
+    pass
+
+
 class GameState(Board):
     """
     State storage for the game, keeps a store of everything it has received.
     """
 
+    FILE_GLOB = '/media/usb?/zone-?'
+    FILE_REGEX = re.compile('zone-(\d)')
+
     # define the name od the board
     board_type_id = 'game'
     create_on_startup = True
 
-    def __init__(self):
-        super().__init__({})
-        self.state = {'zone': 0, 'mode': 'development'}
-
     @classmethod
     def name(cls, node):
         return 'state'
 
-    def command(self, cmd):
-        self.state.update(cmd)
+    def find_zone(self):
+        for candidate_path in glob.glob(self.FILE_GLOB):
```

Comment:
I personally prefer using `glob.iglob` when reading the FS like this

Reply:
Interesting, I'm somewhat surprised that that didn't get removed when most lists became iterators. b64d58e